components/31-molecules/checkboxes/README.md

```diff
@@ -0,0 +1,26 @@
+# Checkboxes
+
+## When to use this component
+
+Use the checkboxes component to help users:
+
+* Select one or more options from a list a short list of options.
+* Toggle a single option on or off.
+
+## When not to use this component
+
+Do not use the checkboxes component when you need to help users select one or more options from a long list of options. In this cases, use <a href="{{path './checkboxes-with-filter.html'}}">checkboxes with filter component</a> instead.
+
+Do not use the checkboxes component when users should only be able to select one option and no more. In this case, use the <a href="{{path './radios.html'}}">radios component</a> instead.
+
+## How it works
+
+* Checkboxes are grouped in a fieldset.
+* The fieldset uses a legend to present a description for the checkboxes. This is typically a question.
```

Comment:
Not sure what you mean exactly with 'This is typically a question'.. If this is a specific UX-related guideline, you might want to specify why the use of a question is desired. 

If not, I'd leave it out, because the legend might as well be something like 'Personal information' or 'Legal information'.

Reply:
Indeed, in case of checkboxes the legend is typically a question to which the checkboxes are possible answers.